requirements_dev.txt

```diff
@@ -1,9 +0,0 @@
-twine
```

Comment:
@septimit We are testing this package with dbt-utils tests, is this the reason for removing this file ?

Reply:
Correct, I propose we only use the official tests from dbt-utils and thus removing the custom test suite. This file does not need to be here anymore.